homeassistant/components/plugwise/climate.py

```diff
@@ -67,17 +67,18 @@ def __init__(
             self._attr_preset_modes = presets
 
         # Determine hvac modes and current hvac mode
-        self._attr_hvac_modes = [HVACMode.HEAT]
+        # Add HVACMode.OFF for Google Home support
```

Comment:
```suggestion
```

Is this to better represent the device in Home Assistant, or it only for Google Home?

Reply:
Without the HVACMode `off` being there as an option, it's not possible to change the thermostat-mode in Google Home.

I think this is true as a general rule for all thermostats present in HA.

Reply:
When mode `off` is present, in my case I see three options: "off", "heat" and "other" (not auto) in Google Home.

Reply:
FYI, I cannot change a thermostat-setpoint from Google Home so there are more incompatibilities between HA and Google Home :)